packages/playground/src/components/contracts_list/contracts_table.vue

```diff
@@ -118,8 +153,8 @@ import { ContractStates, type GridClient } from "@threefold/grid_client";
 import type { NodeStatus } from "@threefold/gridproxy_client";
 import type { ContractLock } from "@threefold/tfchain_client";
 import { DeploymentKeyDeletionError, TFChainErrors } from "@threefold/types";
-import { defineComponent, type PropType, type Ref, ref } from "vue";
-import { capitalize } from "vue";
+import { computed, defineComponent, type PropType, type Ref, ref } from "vue";
+import { capitalize, onMounted } from "vue";
```

Comment:
Could be used in the same line
```suggestion
import { capitalize, onMounted, computed, defineComponent, type PropType, type Ref, ref } from "vue";
```